app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -154,37 +153,58 @@ public void onCreate(Bundle savedInstanceState) {
 
         settings = App.getInstance().getSettings();
 
-        if(settings.isFullscreenArticleView()) {
+        if (settings.isFullscreenArticleView()) {
```

Comment:
it would be nice to have a pull request without all the whitespace changes. Otherwise it is hard to review.
Of course, we can have whitespace changes, but please in a different PR.

Reply:
sorry .. It looks like it came from using the reformat function of the Android studio.